scripts/build-json/build-page-json.js

```diff
@@ -17,7 +17,7 @@ const writeToFile = (propertyName, json) => {
     }
   };
 
-  const dest = path.join(process.cwd(),'packaged/html/elements', `${propertyName}.json`);
+  const dest = path.join(process.cwd(),'packaged/html/reference/elements', `${propertyName}.json`);
```

Comment:
I should have made a point about this before but the use of `/`, will that work for contributors on Windows?

Reply:
I just pushed a commit which should remove this bit of hardcoding and just ensure we use the same directory structure under /packaged as we found under /content. Although I'm not particularly in love with all the path-fiddling going on in there, and there's probably a much nicer way to deal with that.

Reply:
Yes, it's a fiddly at the moment. 
Perhaps because we can't see it just yet but there must be some nifty function that we can use/write that just takes care of all of this in one sweep. Basically, something that changes the root and takes care of creating the sub-directories etc.